actors/datacap/src/lib.rs

```diff
@@ -438,6 +459,19 @@ where
     }
 }
 
+trait AsActorResult<T> {
+    fn to_actor(self) -> Result<T, ActorError>;
```

Comment:
This name was confusing to me during review.  I get what the instantiation does but not where the intention of the name comes from.  At least to me`to_actor_result` or `to_actor_return` would be clearer.

Reply:
I was trying to find something concise, I really want `into` but can't implement `From` here.